boards/arm/serpente/serpente_defconfig

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_SAMD21=y
+CONFIG_SOC_PART_NUMBER_SAMD21E18A=y
+CONFIG_BOARD_SERPENTE=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CONSOLE=y
+CONFIG_GPIO=y
+CONFIG_PINMUX=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_USB=y
+CONFIG_USE_DT_CODE_PARTITION=y
```

Comment:
FYI, better support for programming has recently landed.  See https://github.com/zephyrproject-rtos/zephyr/pull/27061 for how I'm enabling it on other SAMD21 boards.

There's no rush though as it needs #27153 to land and a new SDK release for 'west flash' to just work.

Reply:
I'll subscribe to #27153 and either change the config lines in this PR or in a follow-up PR, depending on how fast 27153 gets merged.

(I'm already in love with the upcoming changes. No more double-tapping the tiny SMD switch to reboot into bootloader!)